README.md

```diff
@@ -268,6 +269,7 @@ npm install --save \
 
 There are several parts to React Router, so it is recommended to read through the documentation available here: https://github.com/reactjs/react-router/tree/master/docs#table-of-contents
 
+[](#css-bundling)
```

Comment:
GitHub automatically adds these IDs to header fields, no need to include this. Also I noticed this is under "Libraries" rather than "General". I see this as a general purpose thing. What do you think?